iac-common/src/test/java/org/sonar/iac/common/testing/IacTestUtils.java

```diff
@@ -41,7 +41,7 @@ private IacTestUtils() {
   }
 
   public static String code(String... lines) {
-    return StringUtils.join(lines, System.getProperty("line.separator"));
+    return StringUtils.join(lines, "\n");
```

Comment:
Why this change?

Reply:
Because it was not compatible with `TreeModelTest` (a basis of `BicepTreeModelTest`).
It failed, but only on Windows machines :-( 
See: https://github.com/SonarSource/sonar-iac/blob/947e23be421244f38c6867d8d19e044438eebccf/iac-common/src/test/java/org/sonar/iac/common/testing/TreeModelTest.java#L117